### ISSUE TITLE
Confused by 'Add File' behavior

### ISSUE THREAD

**Issue by bryphe** [hoschi/yode#3]
This could just be me - but when I tested the 'Add File' functionality, I expected it see the file on the workspace.

I thought that case just wasn't working - took me a minute to discover that my file was now available under "Open File Editor" 😄 

Great work, it's a really cool concept and neat to see it live!

**Comment by hoschi** [comment]
Good point! I use only the files which are there, but wanted a way people play with their own stuff/use cases.
